cfgov/cal/admin.py

```diff
@@ -28,21 +30,23 @@ def start_date(self, obj):
         return obj.dtstart
     start_date.short_description = 'Start Date'
 
+
 class CFPBPDFFileAdmin(admin.ModelAdmin):
     ordering = ['-id']
 
+
 class CFPBCalendarImportICSFileAdmin(admin.ModelAdmin):
 
     def changelist_view(self, request, extra_context=None):
         context = {}
         context.update(extra_context or {})
-        context.update({ 'icsfile': None,})
-        context.update({ 'calendar_record': None,})
-        context.update({ 'processed_events': None,})
+        context.update({'icsfile': None, })
```

Comment:
Why these changes to add extra spaces after the last comma? That doesn't trigger a warning for me with flake8. See [PEP8 Pet Peeves](https://www.python.org/dev/peps/pep-0008/#pet-peeves).